dataprofiler/profilers/profiler_options.py

```diff
@@ -958,6 +968,25 @@ def __init__(
         self.cms_relative_error = cms_relative_error
         self.cms_max_num_heavy_hitters = cms_max_num_heavy_hitters
 
+    def __str__(self) -> str:
+        """
+        Return a human friendly consumable output in string form.
+
+        :vartype dict_string: dict
+        :return: str of the option properties
+        :rtype: str
+        """
+        dict_string: dict = {"CategoricalOptions": []}
+        for iter_option in [
+            a
+            for a in dir(self)
+            if not a.startswith("__") and not callable(getattr(self, a))
+        ]:
+            dict_string["CategoricalOptions"].append(
+                {str(iter_option): str(getattr(self, iter_option))}
+            )
+        return json.dumps(dict_string, indent=4)
+
```

Comment:
might be good to find a way to abstract this a bit more so this ends up in `BaseOption` 90%+ of this code is repeat just with string changes: so I think there is room to make this DRY-er